src/frontend/src/views/signUp.vue

```diff
@@ -0,0 +1,128 @@
+    <div id="signUpForm">
+      <v-form>
+        <v-container>
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="email"
+                :rules="[rules.required]"
+                label="Email address"
+                required
+              ></v-text-field>
+            </v-col>
+            <v-btn
+              :disabled="!email"
+              id="checkEmail"
+              elevation="1"
+              color="primary"
+            >
+              check
+            </v-btn>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="password"
+                :append-icon="show1 ? 'mdi-eye' : 'mdi-eye-off'"
+                :rules="[rules.required, rules.minPw]"
+                :type="show1 ? 'text' : 'password'"
+                name="input-10-1"
+                label="Password"
+                hint="At least 8 characters"
+                counter
+                @click:append="show1 = !show1"
+              ></v-text-field>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="passwordCheck"
+                :append-icon="show2 ? 'mdi-eye' : 'mdi-eye-off'"
+                :rules="[rules.required, rules.minPw, rules.pwCheck]"
+                :type="show2 ? 'text' : 'password'"
+                name="input-10-1"
+                label="Password Check"
+                hint="Enter your password once more."
+                counter
+                @click:append="show2 = !show2"
+              ></v-text-field>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="nickname"
+                :rules="[rules.minName]"
+                label="Nickname"
+                required
+              ></v-text-field>
+            </v-col>
+          </v-row>
+          <div id="signUpButton">
+            <v-row align-self>
+              <div class="signUpBtn">
+                <v-btn color="primary">
+                  Sign Up
+                </v-btn>
+              </div>
+              <div class="signUpBtn">
+                <v-btn color="error">
+                  Cancel
+                </v-btn>
+              </div>
+            </v-row>
+          </div>
+        </v-container>
+      </v-form>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  data() {
+    return {
+      show1: false,
+      show2: false,
+      email: "",
+      password: "",
+      passwordCheck: "",
+      nickname: "",
+
+      rules: {
+        required: input => !!input || "Required.",
+        minPw: password => password.length >= 8 || "Min 8 characters",
+        minName: nickName => nickName.length >= 6 || "Min 6 characters",
+        pwCheck: passwordCheck =>
+          this.password === passwordCheck || "Password mismatch"
+      }
+    };
+  }
+};
+</script>
+<style>
```

Comment:
```suggestion
<style scoped>
```
<style> 태그를 그대로 사용하게 컴포넌트가 달라도(singUp.vue 파일이 아니더라도)
 class명이 같으면 동일한 CSS의 영향을 받아요!  
특히 위처럼 h1 태그로 적용시킨 css 같은 경우에는 다른 컴포넌트 h1 태그도 전부 css가 달라집니다!  
style 태그에 scoped 속성을 주면 해당 컴포넌트에만 css를 적용할 수 있슴니다. 😃